pkg/apis/config/feature_flags_test.go

```diff
@@ -287,6 +287,37 @@ func TestCheckWarnResourceVerificationMode(t *testing.T) {
 	}
 }
 
+// TestCheckAlphaOrBetaAPIFields validates CheckAlphaOrBetaAPIFields for alpha, beta, and stable context
+func TestCheckAlphaOrBetaAPIFields(t *testing.T) {
```

Comment:
Thanks for the PR @pritidesai this looks good.

Just one NIT question, curious why we would explicitly check this is not Stable instead of testing if the intended flags are set correctly eg. verify stable for `config.EnableStableAPIFields(ctx)`?

Reply:
thank you @JeromeJu for the review 👍 

I am not sure if I followed your question 😜 

This is a test function to validate `CheckAlphaOrBetaAPIFields` i.e. return true if `enable-api-fields` set to either `alpha` or `beta` and false otherwise. The test cases below include testing for all three flags.

Reply:
I do agree that for `CheckAlphaOrBetaAPIFields` this makes sense and covers all the api feature flags. 

I was curious at first sight about the naming for `CheckAlphaOrBetaAPIFields` but now I think this is fine as an intermediate stage to beta.